lint.go

```diff
@@ -210,6 +210,7 @@ func (f *file) lint() {
 	f.lintTimeNames()
 	f.lintContextKeyTypes()
 	f.lintContextArgs()
+	f.lintDepreciatedConstants()
```

Comment:
Rename Depreciated as Deprecated.

Depreciate means to lower in value. Deprecated means to express disapproval of.